service/src/main/java/com/commercetools/util/ClientConfigurationUtil.java

```diff
@@ -0,0 +1,83 @@
+package com.commercetools.util;
+
+import io.sphere.sdk.client.BlockingSphereClient;
+import io.sphere.sdk.client.QueueSphereClientDecorator;
+import io.sphere.sdk.client.RetrySphereClientDecorator;
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.client.SphereClientConfig;
+import io.sphere.sdk.client.SphereClientFactory;
+import io.sphere.sdk.retry.RetryAction;
+import io.sphere.sdk.retry.RetryPredicate;
+import io.sphere.sdk.retry.RetryRule;
+
+
+import javax.annotation.Nonnull;
+import java.time.Duration;
+import java.util.Collections;
+import java.util.List;
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+
+import static io.sphere.sdk.http.HttpStatusCode.BAD_GATEWAY_502;
+import static io.sphere.sdk.http.HttpStatusCode.GATEWAY_TIMEOUT_504;
+import static io.sphere.sdk.http.HttpStatusCode.SERVICE_UNAVAILABLE_503;
+
+public final class ClientConfigurationUtil {
+    private static final long DEFAULT_TIMEOUT = 10;
+    private static final TimeUnit DEFAULT_TIMEOUT_TIME_UNIT = TimeUnit.SECONDS;
+    private static final int RETRIES_LIMIT = 20;
```

Comment:
I think 20 is a bit to high, I would start with 5 (like in this example: https://commercetools.github.io/commercetools-jvm-sdk/apidocs/index.html?io/sphere/sdk/client/RetrySphereClientDecorator.html). We can finetune it later if needed

Reply:
https://github.com/commercetools/commercetools-payone-integration/pull/301/commits/04ca6848fdd5adaa07db7fcf287fb6fb5fda832e

Reply:
Done.
https://github.com/commercetools/commercetools-payone-integration/pull/301/commits/0521665989b8fdf783c0ccdf449dd82bfb9bdf6f